src/main/java/net/snowflake/ingest/streaming/internal/ParquetRowBuffer.java

```diff
@@ -100,7 +100,7 @@ public void setupSchema(List<ColumnMetadata> columns) {
    * @param column column metadata
    * @return column parquet type
    */
-  private Type getColumnParquetType(ColumnMetadata column, int id) {
+  protected Type getColumnParquetType(ColumnMetadata column, int id) {
```

Comment:
Ok, I changed it to 'package-private' access.

Reply:
Since it touches `metadata` member variable, I would leave the `getColumnParquetType` (renamed to `generateColumnparquetType`) in `ParquetRowBuffer.java`.